HCK01_2022_Virtual/README.md

```diff
@@ -34,7 +34,8 @@ interested in collaborating to work on relevant topics.
 
 - **Registration:** Fees: none, it's free! Use [this form](https://forms.gle/eJEf7yQq4UeSc1zF9) to register. Deadline: April 2nd.
 
-- **Communication:** to receive information about this and future events please join the [Hackathon Mailing List](https://groups.google.com/g/brain_straight_hackathon_announcements).
+- **Communication:** to receive information about this and future events please join the [Hackathon Mailing List](https://groups.google.com/g/brain_straight_hackathon_announcements). Join us on Monday, March 21st, Monday 21 at 12PM Pacific Time
+[on this Zoom link](https://alleninstitute-org.zoom.us/j/97684575561?pwd=QVZSK3VEcXlEL3c2QW13b2c3U3pvdz09) for a hackathon pre-meeting meet-up for participants to solidify plans, meet each other, and communicate with the organizer insights of what you want to do.
```

Comment:
and communicate with the organizer insights of what you want to do

->

and communicate with the organizers

Reply:
:+1: